clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/config/ElasticSearchConfigProperties.java

```diff
@@ -57,4 +65,12 @@ public int getConnectionTimeout() {
   public void setConnectionTimeout(int connectionTimeout) {
     this.connectionTimeout = connectionTimeout;
   }
+
+  public void setSingleMappingType(boolean singleMappingType) { this.singleMappingType = singleMappingType; }
```

Comment:
Usually we break this into multiple lines

Reply:
Ha, I thought IntelliJ had just collapsed the functions; fixed